apps/notification/schema.py

```diff
@@ -41,12 +53,45 @@ def get_custom_queryset(queryset, info, **kwargs):
         return get_user_notification_qs(info)
 
 
+class AssignmentType(DjangoObjectType):
+    class Meta:
+        model = Assignment
+
+    project = graphene.Field(ProjectDetailType)
+    content_type = graphene.String(required=False)
+    object_id = graphene.ID(required=True)
+    lead_type = graphene.Field(LeadDetailType)
+    entry_type = graphene.Field(EntryType)
+
+    @staticmethod
+    def get_custom_queryset(queryset, info, **kwargs):
+        return get_user_assignment_qs(info)
+
+    @staticmethod
+    def resolve_content_type(root, info):
+        return root.content_type.model
+
+    @staticmethod
+    def resolve_lead_type(root, info):
+        return Lead(root.object_id) if root.content_type.model == 'lead' else None
+
+    @staticmethod
+    def resolve_entry_type(root, info):
+        return Entry(root.object_id) if root.content_type.model == 'entry' else None
```

Comment:
All this data are from another table.

We need to define data loaders for these.

The easiest way will be to create a nested object with `content_type,lead_type,entry_type`. And define the data loader at **AssignmentType** so that the data loader can `content_type,lead_type,entry_type` for each assignment with single data loader.